e2e/tests/toolbarMenuOverflowWrapped-test.ts

```diff
@@ -0,0 +1,58 @@
+import { selectors, itemsCount } from './toolbarMenuOverflowWrapped-example'
+
+const toolbarItem = (index: number) => `.${selectors.toolbarItem}:nth-child(${index + 1})`
+const toolbarItemWrapped = (index: number) =>
+  `.${selectors.toolbarItemWrapper}:nth-child(${index + 1}) .${selectors.toolbarItem}`
+const toolbar = `.${selectors.toolbar}`
+
+describe('Toolbar menu overflow with wrapped first item', () => {
```

Comment:
just small observation we have the same naming for the describe as well in the "toolbarMenuOverflow-test.ts"